src/reducers/index.js

```diff
@@ -62,6 +65,11 @@ const reducer = (state = initialState, action) => {
         newState.constants = [...new Set(newConstants)]
         return newState
 
+      case ADD_RELATIONS:
+        const newRelations = newState.relations.concat(action.payload)
+        newState.relations = _.uniqWith([...newRelations], _.isEqual)
```

Comment:
With underscore.js, you can just do `_.uniq(newRelations)`

Reply:
That didn't seem to work, so we used uniqWith

Reply:
getting : Uncaught (in promise) ReferenceError: _ is not defined for _.iteratee('name')
 after i changed it to the line you recommend